### PR TITLE
feat(wapes-drop): update text and labels for upcoming drop

### PR DESCRIPTION
**Pull request by padala**
[Associated Notion Card](link here)

-----

## 1. Pull request checklist

- [ ] Notion card has been moved to the Code Review column
- [ ] Notion card has a link to this PR
- [ ] A reviewer has been assigned to the Notion card

### CODE REVIEW

File: src/containers/flows/MintDropNFT/steps/Info/Info.tsx [hunk 1]
```diff
@@ -176,25 +175,25 @@ const Info = (props: InfoProps) => {
 				disablePictureInPicture
 				controlsList="nodownload noremoteplayback noplaybackrate nofullscreen"
 				poster={
-					'https://res.cloudinary.com/fact0ry/video/upload/so_0/c_fit,h_426,w_672/v1651090385/zns/kicks-s2-mint-main.jpg'
+					'https://res.cloudinary.com/fact0ry/video/upload/so_0/c_fit,h_426,w_672/v1670257740/zns/wapes-mint-main.jpg'
```
Comment (domw30) [inline src/containers/flows/MintDropNFT/steps/Info/Info.tsx:178]: This looks very cool!

Comment (domw30) [inline src/containers/flows/MintDropNFT/steps/Info/Info.tsx:178]: This looks very cool!

**Review by domw30 — COMMENTED** [review]
Nice Subbu. I tested the mint flow as far as I could. If I can get on Goerli mintlist to try out the mint that would be great, but from what I can see all looks good. One thing I picked up below.

On banner click, open modal and open zine are both actioned. 

https://user-images.githubusercontent.com/39112648/207152259-66b49c1d-e280-4f7f-acc8-9d5a5492b3b8.mov

File: src/containers/flows/Raffle/index.tsx [hunk 2]
```diff
@@ -225,8 +225,8 @@ const RaffleContainer = ({ setClaimDropStage }: RaffleContainerProps) => {
 					<MintDropNFTBanner
 						title={
 							hasRaffleEnded
-								? 'Claim Your Moto Blessings'
-								: 'Claim Your Moto Blessings'
+								? 'The Wapes are Gathering…'
```
[the remark this answers is not in the document]
Comment (padala) [reply src/containers/flows/Raffle/index.tsx:228]: yeah left it as we might need the raffle in future drops. so leaving it as is

File: src/components/MintPreview/MintPreview.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ const MintPreview = (props: MintPreviewProps) => {
 		const parsedLink = ROUTES.MARKET + '/' + getNetworkZNA(link);
 
 		// Check for Claim domain name
-		const claimDomainName = nft.zNA === ZNA.CLAIM_NFT_DOMAIN_ROOT;
+		const claimDomainName = nft.zNA === ZNA.MINT_NFT_DOMAIN_ROOT;
```
[the remark this answers is not in the document]
Comment (padala) [reply src/components/MintPreview/MintPreview.tsx:61]: Will rename it to be generic

**Review by akshatmittal — APPROVED** [review]
Approving changes on request.

**Pull request merged**